### PR TITLE
Issue 8: Add Loss Function

### PR DESCRIPTION
**Pull request by cris-j-dev**

### CODE REVIEW

File: model/loss.py [hunk 1]
```diff
@@ -1,5 +1,85 @@
...
+import torch.nn as nn
+from torch.autograd import Variable
+import numpy as np
 
 
 def nll_loss(output, target):
     return F.nll_loss(output, target)
+
+
+class Loss():
```
Comment (ssaru) [inline model/loss.py:12]: `object` class를 만들어서 Loss 함수를 만드신 이유가 있으신가요?

File: model/loss.py [hunk 2]
```diff
@@ -1,5 +1,85 @@
...
+        self.iou_loss = None
+        self.cls_loss_loss = None
+
+        self.label_name = ('')
+        self.anchors = np.array([], dtype=np.float)
+
+        self.num_classses = len(self.label_name)
+        self.num_anchors = len(self.anchors)
+
+    def forward(self, data, im_data, gt_boxes=None, gt_classes=None, dontcare=None, size_index=0):
```
Comment (ssaru) [inline model/loss.py:34]: 여기서 forward가 네트워크의 forward를 포함하는 거죠?

File: model/loss.py [hunk 3]
```diff
@@ -1,5 +1,85 @@
...
+        iou_pred_np = iou_pred.data.cpu().numpy()
+
+        _boxes, _ious, _classes, _box_mask, _iou_mask, _class_mask = _build_target(bbox_pred_np,
+                                                                                   gt_boxes,
+                                                                                   gt_classes,
+                                                                                   dontcare,
+                                                                                   iou_pred_np,
+                                                                                   size_index)
+
+        _boxes = Variable(torch.from_numpy(Variable(torch.from_numpy(_boxes).type(torch.FloatTensor), volatile=False)))
```
Comment (ssaru) [inline model/loss.py:70]: `Variable`은 deprecated아니예요?!

**Review by ssaru — APPROVED** [review]

File: model/loss.py (hunk 1 above)
Comment (visionNoob) [reply model/loss.py:12]: 저도 궁금합니다. 
yolov1 같은 경우에는 아래와 같이 작성했습니다. 
https://github.com/insurgent92/YOLOv1/blob/b0cf263ebeb1458668fd7f32e334a51d041bb73a/yolov1.py#L193-L249

File: model/loss.py (hunk 2 above)
Comment (visionNoob) [inline model/loss.py:34]: loss 에서는 forward가 불필요하지 않나요?
